src/test/java/graphql/testrunner/service/TestRunnerServiceTest.java

```diff
@@ -52,4 +62,16 @@ void runTestForCustomClasses() {
         verify(prepareEnvironmentService).prepareJar(eq(job));
         verify(commandExecutorService).executeCommand(eq(command));
     }
+
+    @Test
+    void readResultJson() {
+        List<TestStatistics> testStatistics = new ArrayList<>();
+        try {
+            testStatistics = new ObjectMapper().readValue(new File("result.json"),
+                new TypeReference<List<TestStatistics>>() {});
+        } catch (IOException e) {
```

Comment:
is this actually part of your test? Looks to me like it can be dropped.

Reply:
removed.